ocis-pkg/sync/cache.go

```diff
@@ -8,10 +8,10 @@ import (
 
 // Cache is a barebones cache implementation.
 type Cache struct {
-	entries  sync.Map
-	pool     sync.Pool
 	capacity uint64
 	length   uint64
```

Comment:
// has to be the first words
// in order to be 64-aligned on 32-bit architectures.
capacity, length int64 // access atomically

Reply:
Good point. Added.

Reply:
Thanks 🤩